spacy/ml/models/coref.py

```diff
@@ -56,7 +52,7 @@ def convert_coref_clusterer_inputs(model: Model, X: List[Floats2d], is_train: bo
     def backprop(args: ArgsKwargs) -> List[Floats2d]:
         # convert to xp and wrap in list
         gradients = torch2xp(args.args[0])
-        assert isinstance(gradients, Floats2d)
+        # assert isinstance(gradients, Floats2d)
```

Comment:
Why comment out the `assert`?

Reply:
Same reason as in this other PR - it's breaking with recent changes, though in fact the code is still fine. https://github.com/explosion/spaCy/pull/10875#discussion_r888757693